### DISCUSSION TITLE
v0.8.0

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/http-add-on#1011] [Announcements]
We are happy to release KEDA HTTP Add-on v0.8.0 🎉

### New

- **General**: Add configurable TLS on the wire support to the interceptor proxy ([#907](https://github.com/kedacore/http-add-on/issues/907))
- **General**: Add support for collecting metrics using a Prometheus compatible endpoint or by sending metrics to an OpenTelemetry's HTTP endpoint ([#910](https://github.com/kedacore/http-add-on/issues/910))
- **General**: Propagate HTTPScaledObject labels and annotations to ScaledObject ([#840](https://github.com/kedacore/http-add-on/issues/840))
- **General**: Provide support for allowing HTTP scaler to work alongside other core KEDA scalers ([#489](https://github.com/kedacore/http-add-on/issues/489))
- **General**: Support aggregation windows ([#882](https://github.com/kedacore/http-add-on/issues/882))

### Fixes

- **General**: Ensure operator is aware about changes on underlying ScaledObject ([#900](https://github.com/kedacore/http-add-on/issues/900))

### Deprecations

You can find all deprecations in [this overview](https://github.com/kedacore/http-add-on/labels/breaking-change) and [join the discussion here](https://github.com/kedacore/http-add-on/discussions/categories/deprecations).

- **General**: Deprecated `targetPendingRequests` in favor of `spec.scalingMetric.*.targetValue` ([#959](https://github.com/kedacore/http-add-on/discussions/959))

### Other

- **General**: Align with the new format of Ingress in the example demo ([#979](https://github.com/kedacore/http-add-on/pull/979))
- **General**: Unify loggers ([#958](https://github.com/kedacore/http-add-on/issues/958))

## New Contributors
* @Rohithzr made their first contribution in https://github.com/kedacore/http-add-on/pull/913
* @nikhilchintawar made their first contribution in https://github.com/kedacore/http-add-on/pull/953
* @devopsdynamo made their first contribution in https://github.com/kedacore/http-add-on/pull/929
* @fmarani made their first contribution in https://github.com/kedacore/http-add-on/pull/967
* @JoeyC-Dev made their first contribution in https://github.com/kedacore/http-add-on/pull/979
* @zorocloud made their first contribution in https://github.com/kedacore/http-add-on/pull/927

**Full Changelog**: https://github.com/kedacore/http-add-on/compare/v0.7.0...v0.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/http-add-on/releases/tag/v0.8.0'>v0.8.0</a>.</em>

**Comment by gshhong** [comment]
[http-add-on](https://github.com/kedacore/http-add-on/tree/main)/[config](https://github.com/kedacore/http-add-on/tree/main/config)/[interceptor](https://github.com/kedacore/http-add-on/tree/main/config/interceptor)
/e2e-test/tls
Is the functionality of the following tls currently supported?
Can it be used to intercept and count HTTPS requests and perform scaling operations based on the number of HTTPS requests?
